Calculator/ExpressionParserTests/ExpressionParserTests.swift

```diff
@@ -0,0 +1,30 @@
+import XCTest
+
+final class ExpressionParserTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+    }
+
+    override func tearDownWithError() throws {
+        try super.tearDownWithError()
+    }
+
+    func test_1더하기2빼기3곱하기4로입력된값을_parse에넣으면_각각연산자와_피연산자로_나누어진다() {
+        // given
+        let input = "1+2-3*4"
+        let inputOperand: Array<Double> = [1,2,3,4]
```

Comment:
역시 배열의 타입 선언의 형식을 통일해주세요!

```suggestion
        let inputOperand: Array<Double> = [1, 2, 3, 4]
```
`,` 다음은 한 칸씩 띄워주세요 🙂